proto/api/v1/vuln_mgmt_workload_service.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package v1;
+
+import weak "google/api/annotations.proto";
+import "storage/deployment.proto";
+import "storage/image.proto";
+
+option go_package = "./api/v1;v1";
+option java_package = "io.stackrox.proto.api.v1";
+
+message VulnMgmtExportWorkloadsRequest {
```

Comment:
Let's document this a bit more, i.e. the timeout is in seconds, and also a bit more about the query if possible (i.e. referencing it to some other doc we _hopefully_ have around our query language).

Reply:
I'm not aware of any query doc. But I'm also wary of linking to documentation in the API because docs get outdated fast. But I added an example.

Reply:
There's a doc here: https://docs.openshift.com/acs/4.4/operating/search-filter.html

I think it'd be best to start linking things in the API documentation, it's better than only a partial example.